### PR TITLE
[MIG][13.0] stock_picking_show_return: Migration to v13.0

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa TT21693
@carlosdauden @Tardo Review

### CODE REVIEW

File: stock_picking_show_return/readme/CONTRIBUTORS.rst [hunk 1]
```diff
@@ -0,0 +1,3 @@
+* Pedro M. Baeza <pedro.baeza@tecnativa.com>
+* Shruti Singh <shruti.singh@bizzappdev.com>
+* Ruchir Shukla <ruchir@bizzappdev.com>
```
Comment (Tardo) [inline stock_picking_show_return/readme/CONTRIBUTORS.rst]: Add yourself here...

File: stock_picking_show_return/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#. Go to any picking that has a return
+#. See the new tab "Returns" appears.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/154/11.0
```
Comment (Tardo) [inline stock_picking_show_return/readme/USAGE.rst]: :fire: This will be added by the script

**Review by Tardo — CHANGES_REQUESTED** [review]

**Review by carlosdauden — APPROVED** [review]

**Review by carlosdauden — APPROVED** [review]
Code LGTM

**Comment by rousseldenis** [comment]
@sergio-teruel I think pre-commit is not happy

**Comment by sergio-teruel** [comment]
@rousseldenis jajajajaja the pre commit is driving me crazy!!!

**Comment by rousseldenis** [comment]
> @rousseldenis jajajajaja the pre commit is driving me crazy!!!

When you are used to, it becames transparent and you don't think about formatting anymore :-)

**Comment by sergio-teruel** [comment]
@rousseldenis changes done

**Review by rousseldenis — APPROVED** [review]
Code review

**Pull request merged**